engineering/Testing/QA-best-practices.md

```diff
@@ -0,0 +1,81 @@
+# QA-best-practises
+
+## 1. Purpose of this document
+
+This is a summary of QA practices Dwarvesv uses and recommends to be used. It is not supposed to be a detailed description and sometimes cannot fully be used of all tasks but as an overview of the most important QA processes and a list of good practices that should be used.
+
+## 2. QA practices
+
+Everybody in Dwarvesv team has QA responsibilities, even if there is a named QA manager or QA specialists. To Dwarvesv QA means three things.
+
+1.  The service runs as expected and is considered to be created with good practices.
+2.  The service can be easily and cost efficiently maintained and operated.
+3.  The service can be continuously improved and modified cost efficiently
```

Comment:
Missing dot at the end of the sentence, please add into it

Reply:
added